go/tools/macrobench/config.go

```diff
@@ -0,0 +1,123 @@
+/*
+ *
+ * Copyright 2021 The Vitess Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * /
+ */
+
+package macrobench
+
+import (
+	"errors"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"github.com/vitessio/arewefastyet/go/mysql"
+	"strings"
+)
+
+// MacroBenchConfig defines a configuration used to execute macro benchmark.
+// For instance, the MacroBench method uses MacroBenchConfig.
+type MacroBenchConfig struct {
```

Comment:
nit: code styling thing, the name starts with package name. This can be named just Config

Reply:
Changed the name in 4666fa63a55c35c85a879faa2ea2879cd7a6ae09. Following the same thought, I changed `MacroBench` method to `Run`.